torch_tvm/remove_dropout.h

```diff
@@ -0,0 +1,5 @@
+#pragma once
+
+#include <torch/csrc/jit/ir.h>
+
+TORCH_API void RemoveDropout(std::shared_ptr<torch::jit::Graph>& graph);
```

Comment:
Feels like too specific to be a standalone file.

Reply:
True. But as of now it cannot quite merge into some other files. Maybe I can do generic graph_utils or something like that?

Reply:
seems fine for now (fuse linear does the same thing)

maybe we can do an optimize.h header for all of these in another diff